packages/vaadin-tabs/src/vaadin-tabs.d.ts

```diff
@@ -1,9 +1,9 @@
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 import { ListMixin } from '@vaadin/vaadin-list-mixin/vaadin-list-mixin.js';
```

Comment:
**nit:** Sort the mixins in alphabetical order.

Reply:
Done.